app/upgrades/signetlaunch/README.md

```diff
@@ -2,3 +2,44 @@
 
 This folder contains a software upgrade for testing purposes.
 DO NOT USE IN PRODUCTION!
+
```

Comment:
There is one example of shell script running this upgrade with load of BTC Headers at https://github.com/babylonlabs-io/babylon-deployment/blob/main/deployments/local/bbn-start-and-upgrade-signet-launch.sh

Did not linked in the README.md because the babylon-deployment repo is private